funnel/models/event.py

```diff
@@ -163,6 +177,15 @@ def update_from_dict(cls, updated_participant):
         else:
             return None
 
+    @classmethod
+    def add_from_list(cls, space, filename="", events=[]):
+        """
+        Expected CSV format (with header) : name, email, phone, twitter, company
+        """
+        for row in csv_to_rows(filename):
+            participant_dict = {'events': events, 'fullname': row[0], 'email': row[1], 'phone': row[2], 'twitter': row[3], 'company': row[4]}
+            cls.get(space, participant_dict.get('email'), participant_dict, create=True)
+
```

Comment:
This entire method is making me uncomfortable:

1. Dealing with writeable files on what will be a read-only filesystem in future deployments
2. Hardcoding column order